force-app/main/default/objects/Contact/Contact.object-meta.xml

```diff
@@ -16,6 +16,7 @@
         <skipRecordTypeSelect>false</skipRecordTypeSelect>
         <type>Flexipage</type>
     </actionOverrides>
+	<compactLayoutAssignment>%%%NAMESPACE%%%Traction_Thrive_Staff_Layout<compactLayoutAssignment>
```

Comment:
this line isnt closed correctly,
should be <compactLayoutAssignment>%%%NAMESPACE%%%Traction_Thrive_Staff_Layout</compactLayoutAssignment>

Reply:
fixed